package.json

```diff
@@ -24,27 +27,38 @@
   "author": "Tommy Chen <tommy351@gmail.com> (http://zespia.tw)",
   "license": "MIT",
   "dependencies": {
-    "bluebird": "^3.2.2",
-    "cuid": "^2.1.4",
-    "graceful-fs": "^4.1.3",
+    "bluebird": "^3.7.2",
+    "cuid": "^2.1.8",
+    "graceful-fs": "^4.2.10",
     "hexo-log": "^3.0.0",
     "is-plain-object": "^5.0.0",
-    "JSONStream": "^1.0.7",
-    "rfdc": "^1.1.4"
+    "JSONStream": "^1.3.5",
+    "rfdc": "^1.3.0"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
+    "@types/bluebird": "^3.5.36",
+    "@types/chai": "^4.3.1",
+    "@types/graceful-fs": "^4.1.5",
+    "@types/jsonstream": "^0.8.30",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^18.0.0",
+    "@types/sinon": "^10.0.11",
+    "@typescript-eslint/eslint-plugin": "^5.29.0",
+    "@typescript-eslint/parser": "^5.29.0",
+    "chai": "^4.3.6",
     "chai-as-promised": "^7.1.1",
-    "eslint": "^8.1.0",
+    "eslint": "^8.18.0",
     "eslint-config-hexo": "^5.0.0",
-    "jsdoc": "^3.4.0",
-    "lodash": "^4.17.15",
-    "minami": "^1.1.1",
+    "jsdoc": "^3.6.10",
+    "lodash": "^4.17.21",
+    "minami": "^1.2.3",
     "mocha": "^10.0.0",
     "nyc": "^15.0.0",
-    "sinon": "^14.0.0"
+    "sinon": "^14.0.0",
+    "ts-node": "^10.8.1",
+    "typescript": "^4.7.4"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=14 <18.5"
```

Comment:
This is a workaround.
The `Error [ERR_STREAM_PREMATURE_CLOSE]: Premature close` error occurs with Node `18.6`. I don't know this problem is a bug or a spec change.

Reply:
IMHO we should delete `<18.5` before release the new version (`5.0.0`)

Reply:
Deleted